Automattic/jetpack
# IDC: customizing button labels and support URL

#### Changes proposed in this Pull Request:
* add `migrateButtonLabel` and `startFreshButtonLabel` customization paremeters
* add `supportURL` customization parameter to replace the default support page URL

#### Jetpack product discussion
p9dueE-3Xf-p2#comment-7066
p9dueE-3Xf-p2#comment-7067

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Update the [customization plugin](https://gist.github.com/sergeymitr/f4257d64be6282e36b877a18a30d819f).
2. Get the site into the IDC mode and go to any non-Jetpack page in the admin area.
3. The "Migrate" and "Start Fresh" buttons should display the "sample" labels.
4. Switch the browser to the offline mode. Click the "start fresh" button, the error should appear. Click "find out more" link and confirm that it leads to `jetpack.com` main page.
5. Click the "migrate" button, confirm that the "find out more" link also leads to `jetpack.com` main page.
6. Click "Stay in safe mode", click "find out more", confirm it's `jetpack.com` main page.
7. Open the customizer plugin, comment out parameters `mainBodyText` and `nonAdminBodyText`.
8. Reload the IDC screen, the main body text should have the "Learn more about safe mode" link. Confirm that it also leads to `jetpack.com` main page.
9. Log in as a non-admin, repeat the previous step.
